source/OutgoingMessages.Domain/DocumentWriters/RSM012/MeteredDateForMeteringPointCimJsonDocumentWriter.cs

```diff
@@ -65,7 +65,7 @@ private Document ParseFrom(OutgoingMessageHeader header, IReadOnlyCollection<str
         ArgumentNullException.ThrowIfNull(transactions);
 
         var meteredDataForMeasurementSeries = new Collection<MeteredDataForMeasurementSeries>();
```

Comment:
```suggestion
        var meteredDataForMeasurementSeries = new Collection<RenameThisGuy?>();
```
The document is called "NotifyValidatedMeasureData_MarketDocument".
Dont know if it makes sence